melodic/distribution.yaml

```diff
@@ -14811,6 +14811,16 @@ repositories:
       url: https://github.com/anybotics/variant-release.git
       version: 0.1.6-1
     status: maintained
+  ros_vda5050_connector:
```

Comment:
According to CI, this is now alphabetically in the wrong place.  Also, this should match the name of your package, so this really should be `vda5050_connector`.  Same goes for the noetic side.

Reply:
So the thing is that the new name is not [REP-144 compliant](https://www.ros.org/reps/rep-0144.html), particularly the 3rd global rule which says:

```
A package name should not contain ros as it is redundant. Exceptions include core packages and ROS bindings of an upstream library (e.g. moveit_ros)
```

I guess the question is whether this qualifies for an exception?

Reply:
I totally agree... It is difficult making all those marketing/PR people behind me happy.

I now convinced my team to stick with the previous, conform name for the ROS package, but rename the git repository. I hope this is acceptable, even though the package name now differs from the repo name...